vit_jax.ipynb

```diff
@@ -667,351 +498,52 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 15,
+      "execution_count": null,
       "metadata": {
-        "colab": {
-          "base_uri": "https://localhost:8080/"
-        },
         "id": "TSAVpYtP5VaE",
-        "outputId": "1e3aa86c-7257-4a63-ad61-49095f846c2c",
         "vscode": {
           "languageId": "python"
         }
       },
-      "outputs": [
-        {
-          "output_type": "stream",
-          "name": "stderr",
-          "text": [
-            "INFO:absl:Load pre-computed DatasetInfo (eg: splits, num examples,...) from GCS: cifar10/3.0.2\n",
-            "INFO:absl:Load dataset info from /tmp/tmpr88tzuowtfds\n",
-            "INFO:absl:Field info.citation from disk and from code do not match. Keeping the one from code.\n",
-            "INFO:absl:Field info.splits from disk and from code do not match. Keeping the one from code.\n",
-            "INFO:absl:Field info.supervised_keys from disk and from code do not match. Keeping the one from code.\n",
-            "INFO:absl:Field info.module_name from disk and from code do not match. Keeping the one from code.\n"
-          ]
-        }
-      ],
+      "outputs": [],
       "source": [
         "dataset = 'cifar10'\n",
         "batch_size = 512\n",
-        "config = common_config.with_dataset(common_config.get_config(), dataset)\n",
-        "num_classes = input_pipeline.get_dataset_info(dataset, 'train')['num_classes']\n",
```

Comment:
The following 3 lines were migrated after calling `get_data_from_tfds` to populate `ds_train` and `ds_test`.